packages/mui-base/src/useAutocomplete/useAutocomplete.js

```diff
@@ -1028,6 +1021,16 @@ export default function useAutocomplete(props) {
     }
   };
 
+  // Prevent input blur when interacting with the combobox
+  const handleMouseDown = (event) => {
+    if (event.target.getAttribute('id') !== id) {
+      event.preventDefault();
+    }
+    if (event.currentTarget === event.target) {
+      handleInputMouseDown();
+    }
```

Comment:
I think it should also focus on the input as well, otherwise I can't close the popup on blur in this case. I mouse down on the border and then mouse up on one of the options, after that I cannot close the popup when click outside of the autocomplete.


https://user-images.githubusercontent.com/18292247/224472889-4e600990-25cc-4887-8ea1-cdd8eeea92ce.mov